zebra-state/src/service/finalized_state/zebra_db/shielded.rs

```diff
@@ -355,9 +396,30 @@ impl DiskWriteBatch {
             self.zs_insert(&orchard_tree_cf, height, trees.orchard);
         }
 
+        if let Some(subtree) = trees.sapling_subtree {
+            self.zs_insert(&sapling_subtree_cf, subtree.index, subtree.into_data());
```

Comment:
Note: Since this PR writes to the database, it can't merge until after the 1.2.0 release.